packages/react-common/src/moratorium-banner.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+export const CovidMoratoriumBanner: React.FC<{ locale?: string }> = ({
```

Comment:
Can we add a JSDoc-style doc comment here that explains what `locale` is supposed to be?  Just so there's no confusion on the part of the user... also since this package isn't explicitly part of a specific product, it's inherently missing some context, so being extra attentive to documentation is always a good thing!

It might also be helpful, in the documentation, to mention that the component is completely unstyled, i.e. styling it is the responsibility of the caller.